packages/executor/src/services/EntryPointService/constants.ts

```diff
@@ -7,3 +7,5 @@ export const DefaultGasOverheads = {
   bundleSize: 1,
   sigSize: 65,
 };
+
+export const IMPLEMENTATION_ADDRESS_MARKER = "0xA13dB4eCfbce0586E57D1AeE224FbE64706E8cd3";
```

Comment:
I would add this as a configurable parameter to accommodate for chains that support replay protection.

Reply:
it's hardcoded in the smart contract too and not actually used on chain, so I think it's okay to keep it hardcoded